flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/api/EnvironmentSettings.java

```diff
@@ -62,16 +65,17 @@ public class EnvironmentSettings {
     private final ClassLoader classLoader;
 
     private final @Nullable CatalogStore catalogStore;
-
-    private EnvironmentSettings(Configuration configuration, ClassLoader classLoader) {
-        this(configuration, classLoader, null);
-    }
+    private final @Nullable SerializationContext serializationContext;
```

Comment:
```suggestion
    private final @Nullable SqlSerializationContext serializationContext;
```
In the FLIP we agreed on providing a bit more context in the naming.

Reply:
But I'm wondering whether we can come up with a better name: How about `SqlFactory`, `SqlBuilder`, `SqlGenerator`?